src/components/CanvasWrapper.tsx

```diff
@@ -52,6 +55,9 @@ const defaultProps: Partial<CanvasWrapperProps> = {
 export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElement {
   const props = { ...defaultProps, ...inputProps } as Required<CanvasWrapperProps>;
 
+  // Used to delay updates for operations like updating thresholding!
+  // Transitions are cancellable too, so if is interrupted by a new render
+  // the old work will be discarded.
```

Comment:
not sure what this comment refers to - is it in the right place?

Reply:
Oops, this is a holdover from when I was using `useTransition`. I'll remove this.